x/staking/keeper/keeper.go

```diff
@@ -25,6 +25,22 @@ var _ types.ValidatorSet = Keeper{}
 // Implements DelegationSet interface
 var _ types.DelegationSet = Keeper{}
 
+func HistoricalInfoCodec(cdc codec.BinaryCodec) collcodec.ValueCodec[types.HistoricalRecord] {
```

Comment:
Nit, should be in `keys.go` for consistency with bank (https://github.com/cosmos/cosmos-sdk/blob/c25ddd3101a9934d75e127fc7d23cd97bfc1b98c/x/bank/types/keys.go#L39)